test/test_ops.py

```diff
@@ -1000,6 +1002,43 @@ def gen_iou_check(box, expected, tolerance=1e-4):
             gen_iou_check(box_tensor, expected, tolerance=0.002 if dtype == torch.float16 else 1e-3)
 
 
+class TestMasksToBoxes:
+    def test_masks_box(self):
+        def masks_box_check(masks, expected, tolerance=1e-4):
+            out = ops.masks_to_boxes(masks)
```

Comment:
maybe assert that out is of float dtype, since this is the fix you just made in this PR

Reply:
Yes, done.